Source/Core/Common/Crypto/AES.h

```diff
@@ -0,0 +1,14 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <vector>
+
+#include "Common/CommonTypes.h"
+
+namespace AES
+{
+std::vector<u8> Decrypt(const u8* key, u8* iv, const u8* src, u32 size);
```

Comment:
`size` should be a `size_t` (`<cstddef>` will need to be included for its definition)